app/views/article_recruit_mailer/article_recruit_email.text.haml

```diff
@@ -0,0 +1,9 @@
+="Dear Author: Oregon State University Libraries invites you to deposit manuscript of recent publication for open access"
+\-------------------------------------------------
+
+- @publication.web_of_science_source_record.record.titles.each do |t|
+  = t
+  \-------------------------------------------------
+
+Attach manuscript with this link:
+= publication_url(@publication)
```

Comment:
this should be `claim_url(@publication.web_of_science_source_record[:hashed_uid])`